idl/hpp/core_idl/paths.idl

```diff
@@ -25,7 +25,7 @@ module hpp
 
     double length () raises (Error);
 
-    string print () raises (Error);
+    string show () raises (Error);
```

Comment:
I renamed is to `str` in #74.
Could you change it to `str` so that we won't have conflicts when merging.

Reply:
if you fix this in #74 , I'd better remove that from this PR :)

Reply:
If it isn't blocking you, yes, do that please.

Reply:
done :)